File: abc_spec/code_samples/Java/webhooks@{id}/put.java
CheckoutApi api = CheckoutApiImpl.create(sk_XXXX, true, pk_XXXX);

List < String > eventTypes = Arrays.asList("payment_captured", "payment_approved", "payment_declined");

WebhookRequest webhookRequest = WebhookRequest.builder()
  .url("https://example.com/webhooks/updated")
  .eventTypes(eventTypes)
  .build();

WebhookResponse webhook = api.webhooksClient().updateWebhook("wh_ahun3lg7bf4e3lohbhni65335u", webhookRequest).get();

File: abc_spec/code_samples/Node/webhooks@{id}/put.js
import { Checkout } from 'checkout-node-sdk';

const cko = new Checkout('sk_XXXX');

const webhook = await cko.webhooks.updateWebhook('wh_ahun3lg7bf4e3lohbhni65335u', {
	url: 'https://example.com/webhooks/updated',
	active: true,
	headers: {
		authorization: '1234',
	},
	content_type: 'json',
	event_types: ['payment_approved', 'payment_captured'],
});
